var/spack/repos/builtin/packages/py-smartsim/package.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class PySmartsim(PythonPackage):
+    """A workflow library to easily integrate machine learning libraries with
+    high performance computing simulations and applications
+    """
+
+    homepage = "https://www.craylabs.org/docs/overview.html"
+    git = "https://github.com/CrayLabs/SmartSim"
+    pypi = "smartsim/smartsim-0.5.0.tar.gz"
+
+    maintainers("MattToast")
+
+    version("0.5.0", sha256="35b36243dc84af62261a7f772bae92f0b3502faf01401423899cb2a48339858c")
+
+    variant("torch", default=True, description="Build with the pytorch backend")
+
+    depends_on("python@3.8:3.10", type=("build", "run"))
+    depends_on("py-setuptools", type=("build",))
```

Comment:
```suggestion
    depends_on("py-setuptools@39.2:", type=("build",))
```
From `setup.cfg`

Reply:
Excellent!! Thank you for all of the hard work helping shepherd this in!!